controllers/hibp.js

```diff
@@ -36,6 +37,17 @@ async function notify (req, res) {
       throw new Error("Unrecognized breach: " + reqBreachName);
     }
   }
+  // If we don't have a logo of the breach, get the logo.
+  if (!fs.existsSync(`./public/img/logos/${breachAlert.LogoPath}`)) {
```

Comment:
After white-boarding, we're asking Ops to provision us some https://cloud.google.com/storage/ space for these images, and we'll use the GCP node module to write to it.

It may still be helpful to merge this branch though - the emails will start to include the logos.